src/Compilers/Core/Portable/Compilation/Expression.cs

```diff
@@ -104,6 +104,8 @@ public ConditionalChoice(IExpression condition, IExpression ifTrue, IExpression
 
         public OperationKind Kind => OperationKind.ConditionalChoiceExpression;
 
+        public bool HasErrors => false;
```

Comment:
A more correct implementation actually has different behavior per type. This is coming in the next iteration.